### PR TITLE
Docs: remove IRC remnants

### PR DESCRIPTION
**Pull request by Andersson007**
Docs: remove IRC remnants https://forum.ansible.com/t/proposal-consolidating-ansible-discussion-platforms/6812/26?u=andersson007

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @oraNod

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -9,10 +9,9 @@ body:
       value: |
         **Thank you for reporting a bug in Ansible Runner.**
 
-        If you are looking for community support, please reach out on IRC:
-
-          [#ansible](https://web.libera.chat/#ansible)
-          [#ansible-runner](https://web.libera.chat/#ansible-runner)
+        If you are looking for community support, please visit
+        the [Community guide](https://ansible.readthedocs.io/projects/runner/en/latest/community/)
+        for information how to reach out to us.
```
Comment (oraNod) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
        for information on how to get in touch.
```

File: .github/ISSUE_TEMPLATE/documentation_report.yml [hunk 2]
```diff
@@ -10,11 +10,9 @@ body:
...
         **Thank you reporting an issue with Ansible Runner documentation.**
 
-        If you are looking for community support, please reach out on IRC:
-
-          [#ansible](https://web.libera.chat/#ansible)
-          [#ansible-runner](https://web.libera.chat/#ansible-runner)
-
+        If you are looking for community support, please visit
+        the [Community guide](https://ansible.readthedocs.io/projects/runner/en/latest/community/)
+        for information how to reach out to us.
```
Comment (oraNod) [inline .github/ISSUE_TEMPLATE/documentation_report.yml]:
```suggestion
        for information on how to get in touch.
```

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -10,11 +10,9 @@ body:
...
         **Thank you for suggesting a new feature for Ansible Runner.**
 
-        If you are looking for community support, please reach out on IRC:
-
-          [#ansible](https://web.libera.chat/#ansible)
-          [#ansible-runner](https://web.libera.chat/#ansible-runner)
-
+        If you are looking for community support, please visit
+        the [Community guide](https://ansible.readthedocs.io/projects/runner/en/latest/community/)
+        for information how to reach out to us.
```
Comment (oraNod) [inline .github/ISSUE_TEMPLATE/feature_request.yml]:
```suggestion
        for information on how to get in touch.
```

**Comment by Andersson007** [comment]
@oraNod committed, thanks, PTAL

**Review by oraNod — APPROVED** [review]

**Review by Shrews — APPROVED** [review]

**Comment by Andersson007** [comment]
@oraNod @Shrews thanks for reviewing and merging!

**Pull request merged**